CoreDragon/DragonController.h

```diff
@@ -47,6 +47,13 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark Misc
 /*!	Is the user currently dragging something, in this app or any other app? */
 - (BOOL)draggingOperationIsInProgress;
+
+/*! Recalculates active drop targets, e.g. to allow for refreshing cells in scrolling table view. */
+- (void)recalculateActiveDropTargets;
+
+/*! Explicitely invalidates and redraws a highlight for a specific view. */
+- (void)invalidateHighlightForView:(UIView *)view;
+
```

Comment:
Ack, I'm sad these are needed :/ I'd love for these things to be automatic, but I've done a lot of thinking and haven't figured out a better way to do it.

@bogo If you have the time and energy, I'd love to see demo usage of these in one of the demo apps (or in a new demo app). That way we'd have a scaffold to build something better in the future :)